src/main/java/de/jtheb/fh/lagerverwaltung/entities/Compartment.java

```diff
@@ -41,12 +41,14 @@ public int getItemsVolume() {
     }
 
     /**
-     * This Method tests for itemFitsVolume and articleNrFits
+     * This Method tests for itemFitsVolume and itemFits
+     * itemFits tests whether there is an item in Compartment, which fits with articleNr or if the compartment is empty (==null)
+     * itemFitsVolume tests the obvious
```

Comment:
"tests the obvious" NEVER EVER NEVER
nothing is obvious if you need to document it. and still then don't expect everyone to understand it.